test/test_spit_results.py

```diff
@@ -65,3 +65,32 @@ def test__main__success(self, ansible_mod_cls):
 
         assert(mock.call(argument_spec=expected_arguments_spec) ==
                ansible_mod_cls.call_args)
+
+
+class TestSafeAnsibleProperty(unittest.TestCase):
+    def test_missing_fact(self):
+        self.assertEqual(
+            spit_results.safe_ansible_property(
+                {'foo': 'bar'}, 'baz', 'property'),
+            None)
```

Comment:
Nitpick: I think this is better to read:

```python
self.assertEqual(
    split_results.safe_ansible_property(
        {'foo': 'bar'}, 'baz', 'property'),
    None
)
```

Reply:
If you decide to handle this, please also handle for other assertions below.

Reply:
I haven't seen that style before in Python. Did you work on another project that uses it?

It's not the style I'm used to, but I can get used to most styles. What I would most like is consistency, rather than doing different style fixes in different parts of the code. Rather than changing this right now, I would like to leave this code in place and have a broader conversation about whether we want to make our style more consistent. I will bring it up in just a moment.

Reply:
I don't recall any using it from the top of my head. Maybe this is just to me that it is more readable.